kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -225,14 +336,38 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add columns, remove columns, or change the data type of columns 
+   * in your dataset, provided you don't attempt a change that would corrupt 
+   * the data. Kite follows the guidelines in the Avro schema. See 
```

Comment:
Rather than "provided you don't attempt a change that would corrupt the data", this should be "provided you don't attempt a change that is incompatible with written data". This avoids suggesting that data can be corrupted.

The sentence "Kite follows the guidelines in the Avro schema" isn't correct. An Avro schema doesn't contain guidelines, but Avro defines rules for compatible schema evolution.